insights/specs/default.py

```diff
@@ -711,6 +714,38 @@ def pcp_enabled(broker):
     pcs_status = simple_command("/usr/sbin/pcs status")
     php_ini = first_file(["/etc/opt/rh/php73/php.ini", "/etc/opt/rh/php72/php.ini", "/etc/php.ini"])
     pluginconf_d = glob_file("/etc/yum/pluginconf.d/*.conf")
+
+    @datasource(Ps, Hostname, HostContext)
+    def pmlog_summary_file(broker):
+        """
+        Determines the name for the pmlogger file and checks for its existance
+
+        Returns the name of the latest pmlogger summary file if a running ``pmlogger``
+        process is detected on the system.
+
+        Returns:
+            str: Full path to the latest pmlogger file
+
+        Raises:
+            SkipComponent: raises this exception when the command is not present or
+                the file is not present
+        """
+        ps = broker[Ps]
+        if ps.search(COMMAND__contains='pmlogger'):
+            hostname = broker[Hostname].fqdn
+            pcp_log_date = (datetime.date.today() - datetime.timedelta(days=1)).strftime("%Y%m%d")
+            file = "/var/log/pcp/pmlogger/%s/%s.index" % (hostname, pcp_log_date)
+            try:
+                if os.path.exists(file) and os.path.isfile(file):
+                    return file
+            except Exception as e:
+                SkipComponent("Failed to check for pmlogger file existance: {0}".format(str(e)))
+
+        raise SkipComponent
+
+    pmlog_summary = command_with_args(
+        "/usr/bin/pmlogsummary %s mem.util.used mem.physmem kernel.all.cpu.user kernel.all.cpu.sys kernel.all.cpu.nice kernel.all.cpu.steal kernel.all.cpu.idle disk.all.total",
```

Comment:
Please add three more metrics to the `pmlogsummary` command. 

```suggestion
        "/usr/bin/pmlogsummary %s mem.util.used mem.physmem kernel.all.cpu.user kernel.all.cpu.sys kernel.all.cpu.nice kernel.all.cpu.steal kernel.all.cpu.idle disk.all.total mem.util.cached mem.util.bufmem mem.util.free",
```

Reply:
Done